docs/guides/release-notes.md

```diff
@@ -12,10 +12,21 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R3.2.1 (UserConnect-7.2.4996)``
+``Last updated: R3.2.2 (UserConnect-7.2.xxxx)``
 
 ---
 
+## 3.2.2
+### Notes:
```

Comment:
Do we need to include: https://github.com/AviatrixSystems/terraform-provider-aviatrix/commit/cd22478a42a451195ed0c4d2fc94a77831a28e92

Reply:
I still think we should add this  - I think we added it to 8.0 but its also in 7.2